apps/app/src/storage/localstorage/AsyncStorage.ts

```diff
@@ -0,0 +1,20 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+
+export const storeData = async <T>(value: T, key: string): Promise<boolean> => {
+  try {
+    const jsonValue = JSON.stringify(value);
+    await AsyncStorage.setItem(key, jsonValue);
```

Comment:
**question:** What's the reason for converting the value to JSON? Does the library not support non-string types?

Reply:
Nop, it just accept strings. I think this is a a rule for [localstorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) in general